lib/fizzy/parser_expr.cpp

```diff
@@ -394,8 +407,7 @@ parser_result<Code> parse_expr(const uint8_t* pos)
         }
         code.instructions.emplace_back(instr);
     }
-    if (!label_positions.empty())
-        throw parser_error{"code is not balanced (missing end statements)"};
+    assert(label_positions.empty());
```

Comment:
Why is this changed?

Reply:
This is not reachable. The parsing ends when the final `end` is reached. Previously it was also not reachable, it is leftover from some earlier jumps implementation. I left assert as a documentation / target for fuzzing.